iib/workers/tasks/legacy.py

```diff
@@ -36,8 +36,11 @@ def export_legacy_packages(packages, request_id, rebuilt_index_image, cnr_token,
             _verify_package_info(package_dir, rebuilt_index_image)
             _zip_package(package_dir)
             _push_package_manifest(package_dir, cnr_token, organization)
-
-    set_request_state(request_id, 'in_progress', 'Back ported packages successfully pushed to OMPS')
+    payload = {
+        'state': 'in_progress',
+        'state_reason': 'Back ported packages successfully pushed to OMPS',
+    }
+    update_request(request_id, payload, exc_msg='Failed to push package to OMPS')
```

Comment:
`exc_msg` is incorrect in this case since if an exception is raised from `update_request`, then that means updating the request using the IIB PATCH API endpoint failed, not the interaction with OMPS.

Reply:
yup, let me change in next patch